src/library_types.jl

```diff
@@ -2,3 +2,19 @@
       MAJOR_VERSION = Cint(0),
       MINOR_VERSION,
       PATCH_LEVEL)
+
+@enum(cudaDataType,
```

Comment:
Better do `@enum cudaDataType::Cint begin ... end`

Reply:
Also, either we should export these, or dutch the `CUDA_` prefix and refer to the values as `CUDAapi.R_16F`.

Reply:
> Also, either we should export these, or dutch the `CUDA_` prefix and refer to the values as `CUDAapi.R_16F`.

If these are exported, then shall we also export the `libraryPropertyType` so people don't get confused why using similar things from the same file needs different ways. Otherwise, if these are renamed without `CUDA_` prefix, would the `cudaDataType`s be hard to find? :thinking: Is aliasing a good thing in Julia or not, maybe we get both (or put the old names in comment so it becomes easy to find!)?

Reply:
It's not so much about discoverability but rather the redundant `CUDA` prefix (ie. `CUDAapi.CUDA_R_16F`). Getting rid of the `CUDA_` prefix is probably the best, no need to export all those values.